CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+# Contributing
+
+Thank you for taking time to contribute (or at least considering to). CMS-Android is a community project as much as
+it is Crux's. 
+
+Contributions does not necessarily mean you have to modify source code and fix a bug or add a new functionality.
+Reporting a bug using the [Issues](https://github.com/crux-bphc/CMS-Android/issues) tab is also highly appreciated 
+and a good way to start contributing to the project.
+
+It is essential that you know how to use Git as well as GitHub. While the maintainers will be more than happy helping
+you make your first PR, prerequisite knowhow makes it easier for everyone involved. Check this 
+[guide](https://github.com/firstcontributions/first-contributions)  if you've never contributed to a project on 
+GitHub before.
+
+Please note we have a [Code of Conduct](code-of-conduct.md). Please follow it in all your interactions with the project.
+
+When you submit code changes, your submissions are understood to be under the same [MIT License](LICENSE) that covers the 
+project. Feel free to contact the maintainers if that's a concern.
+
+## Moodle Endpoints Used By The App
+
+The Moodle endpoints used by this app have been collected in a [Postman](https://getpostman.com") workspace 
+[Here](https://www.getpostman.com/collections/e2c0439f144f7d3f60ed). You may import this collection into Postman directly using the link. 
+
+## Contributors Pull Request Checklist
+
+1. Fork the repository and branch from `development`.
+
+2. Work on your changes. Make sure you commit as frequently as required. Once you're done making the changes, push the commits
+   to your fork. Checkout the **Commit Messages** section to find guidelines relating to commit messages.
```

Comment:
This shows that you haven't committed to keeping very long lines, which is great! Maybe wrapping lines at 120 chars is a good way to go for this file.

Reply:
True. I've also gone ahead and used link labels instead of inlining them.